scripts/kb_scripts/remove_scsi.py

```diff
@@ -3,32 +3,32 @@
 import re
 
 
-def remove_file_mentions(scsi_file, relative_path, buffer_file_list, tabulation_string) -> list:
-    for scsi_file_line in scsi_file:
-        if scsi_file_line.find(r'"file://') >= 0 and len(relative_path) > 0:
-            scsi_file_line = scsi_file_line[:scsi_file_line.find('//')+2] + \
-                         relative_path + scsi_file_line[scsi_file_line.find('//')+2:]
-        buffer_file_list.append(tabulation_string + scsi_file_line)
-    return buffer_file_list
+def process_scsi(scsi_file_path , structure_name) -> None:
```

Comment:
May be to make tests to check work of this script?

Reply:
No need for tests